.ci/.bump-go-release-version.yml

```diff
@@ -56,6 +56,12 @@ projects:
     overrideGoVersion: "1.17"
     enabled: true
     labels: dependency
+  - repo: elastic-agent-inputs
+    script: .ci/bump-go-release-version.sh
+    branches:
+      - main
+    enabled: true
```

Comment:
```suggestion
    overrideGoVersion: "1.17"
    enabled: true
```

Reply:
Only if `1.17` is the minor Golang version, otherwise it will point to the latest one, so `1.18` now

Reply:
I would like to stay on 1.17 like in Beats.